tests/functionality_tests/localization_visualizer.py

```diff
@@ -180,8 +185,9 @@ def animate(i):
                 new_location = (measurements[0], measurements[1])
 
             elif not is_live and use_ekf:
-                mu_bar = ekf.mu  # temporary placeholder
-                sigma_bar = ekf.sigma  # temporary placeholder
+                arc_lengths = ekf.get_arc_lengths(1,1) # temporary placeholder
```

Comment:
is this something we can vary and test?

Reply:
these arc lengths are dependent on the velocity - im thinking that this is another piece of data that we would have to collect or simulate (im not sure how we would go about simulating it though)

Reply:
I think as long as we have a theta, r, and width, we have a valid arclength (arclength = theta*R, where R = r or r + width. Then we can randomize theta, r, and width to see if the property holds

Reply:
making a note of this for potential testing